src/main/java/edu/wpi/first/gradlerio/wpi/dependencies/tools/WPICppTool.java

```diff
@@ -18,7 +17,7 @@ public class WPICppTool implements Named {
     private final Provider<String> version;
 
     public WPICppTool(Project project, String name, Provider<String> version, String artifactId, Provider<Directory> toolsFolder) {
-        Configuration config = project.getConfigurations().getByName("wpiCppTools");
+        //Configuration config = project.getConfigurations().getByName("wpiCppTools");
```

Comment:
Commented-out code should be removed?

Reply:
I want to keep this for when we finally getting around to adding installation for C++ tools.